recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ test:
     - pyqt_test.py
   imports:
     - PyQt5.QtCore
+    - PyQt5.QtWebKitWidgets
```

Comment:
Please change this one to

    - PyQt5.QtWebKitWidgets  # [linux or win]
    - PyQt5.QtWebEngineWidgets  # [osx]

This is because on macOS we provide WebEngine instead of WebKit